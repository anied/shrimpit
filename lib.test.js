const Shrimpit = require('./lib')

it('should create a snapshot of the files tree and of the unused exports', () => {
  const shrimpit = new Shrimpit(['', '', 'test', '--json'])
  expect(shrimpit.exec()).toMatchSnapshot();
})
