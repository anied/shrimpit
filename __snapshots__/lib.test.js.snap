// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should create a snapshot of the files tree and of the unused exports 1`] = `"{\\"Files tree\\":{\\"test\\":{\\"a\\":{\\"a.js\\":{\\"imports\\":[{\\"location\\":\\"test/b/b.js\\",\\"name\\":\\"test\\",\\"unamedDefault\\":true}],\\"exports\\":[{\\"location\\":\\"test/a/a.js\\",\\"name\\":\\"a\\",\\"unamedDefault\\":false},{\\"location\\":\\"test/a/a.js\\",\\"name\\":\\"c\\",\\"unamedDefault\\":false},{\\"location\\":\\"test/a/a.js\\",\\"name\\":\\"d\\",\\"unamedDefault\\":true}]}},\\"b\\":{\\"b.js\\":{\\"imports\\":[{\\"location\\":\\"test/c/c.js\\",\\"name\\":\\"Cat\\",\\"unamedDefault\\":false},{\\"location\\":\\"test/a/a.js\\",\\"name\\":\\"test\\",\\"unamedDefault\\":true}],\\"exports\\":[{\\"location\\":\\"test/b/b.js\\",\\"name\\":\\"a\\",\\"unamedDefault\\":false},{\\"location\\":\\"test/b/b.js\\",\\"name\\":\\"b\\",\\"unamedDefault\\":false},{\\"location\\":\\"test/b/b.js\\",\\"name\\":\\"default (unamed)\\",\\"unamedDefault\\":true}]}},\\"c\\":{\\"c.js\\":{\\"imports\\":[{\\"location\\":\\"test/a/a.js\\",\\"name\\":\\"a\\",\\"unamedDefault\\":false},{\\"location\\":\\"test/a/a.js\\",\\"name\\":\\"c\\",\\"unamedDefault\\":false},{\\"location\\":\\"test/b/b.js\\",\\"name\\":\\"b\\",\\"unamedDefault\\":false}],\\"exports\\":[{\\"location\\":\\"test/c/c.js\\",\\"name\\":\\"Cat\\",\\"unamedDefault\\":false},{\\"location\\":\\"test/c/c.js\\",\\"name\\":\\"User\\",\\"unamedDefault\\":true}]}}}},\\"Unused exports\\":[{\\"location\\":\\"test/b/b.js\\",\\"name\\":\\"a\\",\\"unamedDefault\\":false},{\\"location\\":\\"test/c/c.js\\",\\"name\\":\\"User\\",\\"unamedDefault\\":true}]}"`;
